frontend/src/components/Schedules.vue

```diff
@@ -0,0 +1,59 @@
+<script setup>
+import { ref, onMounted } from "vue";
+import { defineProps } from 'vue';
```

Comment:
[@vue/compiler-sfc] `defineProps` is a compiler macro and no longer needs to be imported.